sdk/eventhub/azure-eventhubs/azure/eventhub/_common.py

```diff
@@ -200,9 +154,9 @@ def enqueued_time(self):
 
         :rtype: datetime.datetime
         """
-        timestamp = self.message.annotations.get(EventData._PROP_TIMESTAMP, None)
+        timestamp = self.message.annotations.get(PROP_TIMESTAMP, None)
         if timestamp:
-            return datetime.datetime.utcfromtimestamp(float(timestamp)/1000)
+            return utc_timestamp(float(timestamp)/1000)
```

Comment:
This may be a silly nit, but while I figured this was the behavior, it took seeing this delta to be fully convinced (without looking in utils) that utc_timestamp does the right thing.  I would probably keep the naming more similar to the original verb form for a function call, but as with all my style nits, you can override this if there's disagreement.

Reply:
Updated.